src/ubuntu/manifest.json

```
@@ -302,6 +302,19 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/ubuntu/24.04/sdk/amd64",
+              "os": "linux",
+              "osVersion": "noble",
+              "tags": {
+                "ubuntu-24.04-sdk-amd64": {}
```

The tag name (and file path) should reflect its purpose. The `CODEOWNERS` file will also need to be updated to reflect this.

```suggestion
              "dockerfile": "src/ubuntu/24.04/source-build-test/amd64",
              "os": "linux",
              "osVersion": "noble",
              "tags": {
                "ubuntu-24.04-source-build-test-amd64": {}
```